### DISCUSSION TITLE
How can I check and fix the replication of a specific volume?

### DISCUSSION THREAD

**Discussion by Programmeris** [seaweedfs/seaweedfs#4528] [Q&A] [answered]
I have a SeaweedFS 3.37 cluster with lots of volumes on Volume Servers. I need to check and fix the replication of specific volumes by their Id.
`volume.fix.replication,` as far as I understand, affects all volumes in all collections. The maximum that is possible is to select a collection in which replication will be checked and corrected.
Is it possible to run a replication check and fix for a specific id?
Something like `volume.fix.replication --id=[volume id]` ?

**Comment by chrislusf** [comment, marked as answer]
Not supported yet.

**Reply by Programmeris** [reply]
Thank you for the answer!
